guides/common/modules/proc_changing-the-module-stream-for-a-host.adoc

```diff
@@ -10,4 +10,4 @@ You can enable, disable, install, update, and remove module streams from your ho
 . Click the name of the host you want to modify.
 . Click the *Content* tab, then click the *Module streams* tab.
 . Click the vertical ellipsis next to the module and select the action you want to perform.
-You get a REX job notification once the remote execution job is complete.
\ No newline at end of file
+You get a REX job notification once the remote execution job is complete.
```

Comment:
```suggestion
You will get a REX job notification once the remote execution job is complete.
```

Reply:
Here I decided not to apply since we should avoid using future tense where possible. In this conditional, I deemed it not necessary.
https://docs.engineering.redhat.com/display/CCSENABLE/Future+tense